test/development/acceptance/ReactRefreshRegression.test.ts

```diff
@@ -286,10 +286,18 @@ describe('ReactRefreshRegression', () => {
     expect(await session.hasRedbox()).toBe(true)
 
     const source = await session.getRedboxSource()
-    expect(source.split(/\r?\n/g).slice(2).join('\n')).toMatchInlineSnapshot(`
-      "> 1 | export default function () { throw new Error('boom'); }
-          |                                   ^"
-    `)
+    const splittedSource = source.split(/\r?\n/g).slice(2).join('\n')
```

Comment:
We shouldn't split the source here but instead snapshot test the entirety of the result, the splitting is causing the tested line to be wrong, if that is the only part shown in the overlay there's a bug in Turbopack handling